ababot/bot/src/commands/kok/download.rs

```diff
@@ -0,0 +1,156 @@
+use std::path::Path;
+
+use reqwest::multipart::{Form, Part};
+use serenity::{
+    model::prelude::interaction::{
+        application_command::ApplicationCommandInteraction, InteractionResponseType,
+    },
+    prelude::Context,
+};
+use tokio::fs::read;
+
+use super::{types::Annonfile, upload::local_parse};
+
+const URL: &str = "https://api.anonfiles.com/upload";
+
+pub async fn get(ctx: &Context, command: &ApplicationCommandInteraction, file_str: &str) {
+    if let Err(why) = command
+        .create_interaction_response(&ctx.http, |response| {
+            response.kind(InteractionResponseType::DeferredChannelMessageWithSource)
+        })
+        .await
+    {
+        tracing::warn!("Not able to ack interaction: {:?}", why);
+        return;
+    }
+
+    if file_str.is_empty() || file_str.contains('/') {
+        error(ctx, command, "Illegal file type").await;
```

Comment:
Is this a problem with file types?

Reply:
Problem was the ability to fetch arbitrary file from host